drivers/clock_control/Kconfig

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if BT_LL_NRFXLIB
+
+config CLOCK_CONTROL_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller clock control driver"
+	default y
+	select CLOCK_CONTROL_NRF_FORCE_ALT
+
+config CLOCK_CONTROL_NRF_M16SRC_DRV_NAME
```

Comment:
Both device names and properties of clock source should be taken from DTS.

Reply:
This is the problem we faced as well at the beginning when we started working on Zephyr. @pizi-nordic is right in this case. All HW settings shall be taken from DT not from KConfig.

Reply:
:+1: the drivers were written when these values were not yet fully in DTS.